Ryujinx.HLE/HOS/Services/Nv/NvDrvServices/NvHostCtrl/Types/GetConfigurationArguments.cs

```diff
@@ -0,0 +1,34 @@
+﻿using System;
+using System.Text;
+
+namespace Ryujinx.HLE.HOS.Services.Nv.NvDrvServices.NvHostCtrl.Types
+{
+    class GetConfigurationArguments
+    {
+        public string Domain;
+        public string Parameter;
+        public byte[] Configuration;
+
+        public static GetConfigurationArguments FromSpan(Span<byte> span)
+        {
+            string domain        = Encoding.ASCII.GetString(span.Slice(0, 0x41));
+            string parameter     = Encoding.ASCII.GetString(span.Slice(0x41, 0x41));
```

Comment:
```suggestion
            string parameter = Encoding.ASCII.GetString(span.Slice(0x41, 0x41));
```
same for the other var